### PR TITLE
Vm edu 7 event feed component

### PR DESCRIPTION
**Pull request by limoozyn**
Initial component for the event feed.The Titles and dates will be stubbed with faker next PR.
--

**Checklist**

- [x] [YouTrack ticket](https://tutorium.myjetbrains.com/youtrack/issue/EDU-7) created
- [x] ready to review

**Why?**

This rough component will evolve into mature component with tests and API integration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-react": "6.4.1",
     "extract-text-webpack-plugin": "^1.0.1",
+    "faker": "^4.1.0",
```
Comment (roman-holovin) [inline package.json:27]: `faker` is not used anywhere

File: src/containers/main_area/MainArea.js [hunk 2]
```diff
@@ -1,8 +1,59 @@
 import React from 'react'
+import { List, ListItem, ListSubHeader} from 'react-toolbox/lib/list'
+import styles from './MainArea.scss'
+
+function Day(props) {
+  const listEvents = props.dateAndEvents.events.map((event, index) =>
+    <ListItem key={index} caption={event} className={styles.event} />
```
Comment (roman-holovin) [inline src/containers/main_area/MainArea.js]: Don't use `index` as key for dynamic data.

File: src/containers/main_area/MainArea.js [hunk 3]
```diff
@@ -1,8 +1,59 @@
...
+      <List>
+        <ListSubHeader caption={props.dateAndEvents.date} />
+        {listEvents}
+      </List>
+    </ListItem>
+  );
+};
+
+function DaysList(props) {
+  const listDays = props.dates.map((aDate) =>
```
Comment (roman-holovin) [inline src/containers/main_area/MainArea.js]: `aDate` is strange name for a variable, `date` will be sufficient

File: src/containers/main_area/MainArea.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.container__list {
+  margin: 50px auto;
+  max-width: 1024px;
+}
+.event{
+  border-bottom: 1px solid #d8d8d8;
```
Comment (roman-holovin) [inline src/containers/main_area/MainArea.scss]: Use variables

**Review by roman-holovin — CHANGES_REQUESTED** [review]
Rename `MainArea` to `Events`
Refactor `DaysList`, put code into `Events` component
Refactor `Day` component into separate file (`components/Day` directory)

File: package.json (hunk 1 above)
Comment (limoozyn) [reply package.json:27]: I will use it next time to stub events and test them.

**Review by roman-holovin — APPROVED** [review]

**Pull request merged**